data/cloudcraft.json

```diff
@@ -1751,6 +1751,167 @@
+        "tags": [
+          "Teams"
+        ],
+        "summary": "List teams",
+        "description": "List all teams linked to your Cloudcraft account.\n\nThe response is an array of teams. Each entry includes information about the team, such as the team ID, name, and members.",
+        "operationId": "listMyTeams",
+        "responses": {
+          "200": {
+            "description": "OK",
+            "headers": {
+              "Cache-Control": {
+                "schema": {
+                  "type": "string",
+                  "example": "no-cache, no-store"
+                }
+              },
+              "Connection": {
+                "schema": {
+                  "type": "string",
+                  "example": "keep-alive"
+                }
+              },
+              "Content-Length": {
+                "schema": {
+                  "type": "string",
+                  "example": "271"
+                }
+              },
+              "Date": {
+                "schema": {
+                  "type": "string",
+                  "example": "Wed, 14 Nov 2018 21:14:32 GMT"
+                }
+              },
+              "ETag": {
+                "schema": {
+                  "type": "string",
+                  "example": "W/\"10f-XQrUSiTExKF+oTaRV4ZXDCtfrGc\""
+                }
+              },
+              "Vary": {
+                "schema": {
+                  "type": "string",
+                  "example": "Accept-Encoding"
+                }
+              },
+              "X-Content-Type-Options": {
+                "schema": {
+                  "type": "string",
+                  "example": "nosniff"
+                }
+              },
+              "X-Frame-Options": {
+                "schema": {
+                  "type": "string",
+                  "example": "deny"
+                }
+              },
+              "X-XSS-Protection": {
+                "schema": {
+                  "type": "string",
+                  "example": "1; mode=block"
+                }
+              }
+            },
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "properties": {
+                    "id": {
+                      "type": "string",
+                      "description": "The unique identifier of the team.",
+                      "example": "us46e9aa-5806-4cd6-8e78-c22d58602d09"
+                    },
+                    "name": {
+                      "type": "string",
+                      "description": "The name of the team.",
+                      "example": "Example Team Name"
+                    },
+                    "visible": {
+                      "type": "boolean",
+                      "description": "Whether the team is visible to other users.",
+                      "example": true
+                    },
+                    "crossOrganizational": {
```

Comment:
I think we should hide this attribute in the public documentation as we will not not support cross org membership @tjunnone - thoughts?

Reply:
This is an unfortunately named team property that means the members in this team have the team permissions applied globally in the org (across the entire organization, not across different organizations). Since it's there, and also exposed in the UI (for Enterprise plans only), I'm OK with keeping it documented unless we decide to retire the entire feature.